### PR TITLE
[MNT] Removed `FromInt` age transformation

### LINKED ISSUE #636: Remove `FromInt` transformation and supporting only `FromFloat`
This may be simpler and less confusing for the user when they have a column with a mix of float and int ages

### PR DESCRIPTION
**Pull request by rmanaem**
- Closes #636 

## Checklist
_This section is for the PR reviewer_

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`, `[MODEL]`, `[DOC]`) _(see our [Contributing Guidelines](https://neurobagel.org/contributing/CONTRIBUTING#pull-request-guidelines) for more info)_
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [x] Tests pass
- [x] Checks pass

For new features:
- [ ] Tests have been added

For bug fixes:
- [ ] There is at least one test that would fail under the original bug conditions.

## Summary by Sourcery

Enhancements:
- Remove the 'FromInt' age transformation from the store configuration and related test files.

### CODE REVIEW

File: cypress/fixtures/examples/good/ds003653_participant_annotated_1698934398962.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
             },
             "Transformation": {
                 "Label": "integer value",
-                "TermURL": "nb:FromInt"
+                "TermURL": "nb:FromFloat"
```
Comment (surchs) [inline cypress/fixtures/examples/good/ds003653_participant_annotated_1698934398962.json:20]: smart

Comment (surchs) [reply cypress/fixtures/examples/good/ds003653_participant_annotated_1698934398962.json:20]: maybe change the label to "float value" as well

**Review by surchs — APPROVED** [review]
Looks good @rmanaem !

:cook:

File: cypress/fixtures/examples/good/ds003653_participant_annotated_1698934398962.json (hunk 1 above)
Comment (rmanaem) [reply cypress/fixtures/examples/good/ds003653_participant_annotated_1698934398962.json:20]: Ah missed that one

**Pull request merged**